backend/libs/parking/src/parking.resolver.ts

```diff
@@ -5,49 +5,82 @@ import { ParkingService } from "./parking.service";
 import { GqlAuthGuard } from "@vms/auth/guards/GqlAuthGuard.guard";
 
 import { Parking } from "./models/parking.model";
+import { ParkingReservation } from "./models/reservation.model";
 
 @Resolver((of) => Parking)
 export class ParkingResolver {
     constructor(
         private parkingService: ParkingService,
     ) {}
 
-    @UseGuards(GqlAuthGuard)
+    //@UseGuards(GqlAuthGuard)
```

Comment:
Please uncomment these, I'm guessing it's just for the testing purposes, but just when you make your final commit

Reply:
> Please uncomment these, I'm guessing it's just for the testing purposes, but just when you make your final commit

Noted